tests/cli/test_schema_generation.py

```diff
@@ -34,12 +34,12 @@ class SubPipelineComponent(EmptyPipelineComponent):
     )
 
 
-# schema_type is ineritted from SubPipelineComponent
+# schema_type is inerited from SubPipelineComponent
```

Comment:
```suggestion
# schema_type is inherited from SubPipelineComponent
```

Reply:
saw this, took 2 tries, but it's fixed now haha